server-reboot/duffy.py

```diff
@@ -0,0 +1,110 @@
+import json, urllib, subprocess, sys, os, time
```

Comment:
I dont think this file should be included, can't you use the [common-scripts/basic-gluster-duffy.py](https://github.com/nfs-ganesha/ci-tests/blob/centos-ci/common-scripts/basic-gluster-duffy.py) instead? If you made modifications, maybe those can be merged into that one?

Reply:
modification in these are only required for this particular test.  https://github.com/nfs-ganesha/ci-tests/blob/centos-ci/common-scripts/basic-gluster-duffy.py is used by other tests so can not modify it.